Causing issues with Strapi v4.15.0

https://github.com/offset-dev/strapi-calendar/blob/93f0acccf2545ac777f15f57ba15ed9ce6918844/package.json#L29

I think it just needs to be decremented to 5.3.3 which `@strapi/admin@4.15.0` is using 

```
$ npm install @offset-dev/strapi-calendar@0.1.0
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: peak-golf-academy@0.1.0
npm ERR! Found: styled-components@5.3.3
npm ERR! node_modules/styled-components
npm ERR!   styled-components@"5.3.3" from @strapi/admin@4.15.0
npm ERR!   node_modules/@strapi/admin
npm ERR!     @strapi/admin@"4.15.0" from @strapi/strapi@4.15.0
npm ERR!     node_modules/@strapi/strapi
npm ERR!       peer @strapi/strapi@"^4.3.4" from @strapi/admin@4.15.0
npm ERR!       5 more (@strapi/data-transfer, @strapi/plugin-documentation, ...)
npm ERR!   peer styled-components@"^5.2.1" from @strapi/design-system@1.13.0
npm ERR!   node_modules/@strapi/admin/node_modules/@strapi/design-system
npm ERR!     @strapi/design-system@"1.13.0" from @strapi/admin@4.15.0
npm ERR!     node_modules/@strapi/admin
npm ERR!       @strapi/admin@"4.15.0" from @strapi/strapi@4.15.0
npm ERR!       node_modules/@strapi/strapi
npm ERR!         peer @strapi/strapi@"^4.3.4" from @strapi/admin@4.15.0
npm ERR!         5 more (@strapi/data-transfer, @strapi/plugin-documentation, ...)
npm ERR!     peer @strapi/design-system@"1.13.0" from @strapi/helper-plugin@4.15.0
npm ERR!     node_modules/@strapi/admin/node_modules/@strapi/helper-plugin
npm ERR!       @strapi/helper-plugin@"4.15.0" from @strapi/admin@4.15.0
npm ERR!       node_modules/@strapi/admin
npm ERR!         @strapi/admin@"4.15.0" from @strapi/strapi@4.15.0
npm ERR!         node_modules/@strapi/strapi
npm ERR!   17 more (@strapi/helper-plugin, @strapi/design-system, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! @offset-dev/strapi-calendar@"0.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: styled-components@5.3.11
npm ERR! node_modules/styled-components
npm ERR!   peer styled-components@"^5.3.6" from @offset-dev/strapi-calendar@0.1.0
npm ERR!   node_modules/@offset-dev/strapi-calendar
npm ERR!     @offset-dev/strapi-calendar@"0.1.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@offset-dev/strapi-calendar",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "Visual calendar for Strapi",
   "strapi": {
     "name": "calendar",

